tests/robot-cases/Group0-BAT/API_DB.robot

```diff
@@ -49,23 +49,23 @@ Test Case - Edit Project Creation
 # TODO uncomment this after bump up chart API version to v2.0
 # Test Case - List Helm Charts
 #     Harbor API Test  ./tests/apitests/python/test_list_helm_charts.py
-### Test Case - Assign Sys Admin
-###     Harbor API Test  ./tests/apitests/python/test_assign_sys_admin.py
+Test Case - Assign Sys Admin
+    Harbor API Test  ./tests/apitests/python/test_assign_sys_admin.py
 Test Case - Copy Artifact Outside Project
     Harbor API Test  ./tests/apitests/python/test_copy_artifact_outside_project.py
-### Test Case - Robot Account
-###     Harbor API Test  ./tests/apitests/python/test_robot_account.py
-### Test Case - Sign A Image
-###     Harbor API Test  ./tests/apitests/python/test_sign_image.py
+Test Case - Robot Account
+    Harbor API Test  ./tests/apitests/python/test_robot_account.py
+Test Case - Sign A Image
+    Harbor API Test  ./tests/apitests/python/test_sign_image.py
 # TODO uncomment this after making quota work with OCI registry
 # Test Case - Project Quota
 #    Harbor API Test  ./tests/apitests/python/test_project_quota.py
-### Test Case - System Level CVE Whitelist
-###     Harbor API Test  ./tests/apitests/python/test_sys_cve_whitelists.py
-### Test Case - Project Level CVE Whitelist
-###     Harbor API Test  ./tests/apitests/python/test_project_level_cve_whitelist.py
-### Test Case - Tag Retention
-###     Harbor API Test  ./tests/apitests/python/test_retention.py
-### Test Case - Health Check
-###     Harbor API Test  ./tests/apitests/python/test_health_check.py
+Test Case - System Level CVE Whitelist
+    Harbor API Test  ./tests/apitests/python/test_sys_cve_whitelists.py
+Test Case - Project Level CVE Whitelist
+    Harbor API Test  ./tests/apitests/python/test_project_level_cve_whitelist.py
+Test Case - Tag Retention
+    Harbor API Test  ./tests/apitests/python/test_retention.py
+Test Case - Health Check
+    Harbor API Test  ./tests/apitests/python/test_health_check.py
```

Comment:
I recall some cases for LDAP are also commented, can they be uncommented again?

Reply:
LDAP test was added.